frontend/mr-admin-flate/src/components/utbetaling/OpprettDelutbetalingRow.tsx

```diff
@@ -0,0 +1,135 @@
+import { formaterDato, tilsagnTypeToString } from "@/utils/Utils";
+import { TilsagnDto, TilsagnStatus, Totrinnskontroll } from "@mr/api-client-v2";
+import { Alert, Checkbox, Table, TextField } from "@navikt/ds-react";
+import { useState } from "react";
+import { AvvistAlert } from "@/pages/gjennomforing/tilsagn/AarsakerAlert";
+import { DelutbetalingTag } from "./DelutbetalingTag";
+import { formaterNOK } from "@mr/frontend-common/utils/utils";
+import { TilsagnStatusTag } from "./TilsagnTag";
+
+interface Props {
+  id?: string;
+  tilsagn: TilsagnDto;
+  belop: number;
+  frigjorTilsagn: boolean;
+  opprettelse?: Totrinnskontroll;
+  kanRedigere: boolean;
+  onDelutbetalingChange: (d: {
+    id?: string;
+    tilsagnId: string;
+    belop: number;
+    frigjorTilsagn: boolean;
+  }) => void;
+}
+
+export function OpprettDelutbetalingRow({
+  id,
+  tilsagn,
+  belop,
+  frigjorTilsagn,
+  opprettelse,
+  kanRedigere,
+  onDelutbetalingChange,
+}: Props) {
+  const [endretBelop, setEndretBelop] = useState<number>(belop);
+  const [endretFrigjorTilsagn, setEndretFrigjorTilsagn] = useState<boolean>(frigjorTilsagn);
+  const [openRow, setOpenRow] = useState<boolean>(frigjorTilsagn);
+  const [error, setError] = useState<string | undefined>(undefined);
+
+  const godkjentTilsagn = tilsagn.status === TilsagnStatus.GODKJENT;
+
+  function handleOnChange(belop: number, frigjorTilsagn: boolean) {
+    onDelutbetalingChange({
+      id: id,
+      tilsagnId: tilsagn.id,
+      belop: belop,
+      frigjorTilsagn: frigjorTilsagn,
+    });
+  }
+  const cellClass = error && "align-top";
+  return (
+    <Table.ExpandableRow
+      defaultOpen={opprettelse && true}
```

Comment:
Hva er greien med `&& true`? Er det for å kaste til en boolean? I så fall kan du gjøre `!!opprettelse`, eller `Boolean(opprettelse)` istedenfor.